lib/puppet_litmus/rake_tasks.rb

```diff
@@ -87,10 +87,20 @@ def run_local_command(command)
   task :provision_list, [:key] do |_task, args|
     provision_hash = YAML.load_file('./provision.yaml')
     provisioner = provision_hash[args[:key]]['provisioner']
+    failed_image_message = ''
```

Comment:
Should it be a hash of image -> error? Makes it easy to separate the formatting and allows you to provide a sorted list.

Reply:
@ekohl that was the way i had it for my first attempt. It didnt really add much, and this just simplifies the code. We cant use a hash, because there may be more than one entry for a given image. EG i want to spin up 3 centos-7 machines. if in a hash the errors would be masked by each other.

Reply:
Fair enough